### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by Aadi-mone** [booksaw/BetterTeams#239]
The log says something:
https://mclo.gs/Pu91JIo

**Comment by booksaw** [comment]
That should never happen and is a bug that will be fixed.

pasting log here so it does not expire: 

```
1 | [20:44:32] [Server thread/ERROR]: Error occurred while enabling BetterTeams v4.1.3 (Is it up to date?)
-- | --
2 | java.lang.NullPointerException: Cannot invoke "com.booksaw.betterTeams.Team.getName()" because "team" is null
3 | at com.booksaw.betterTeams.integrations.HologramManager.reloadHolo(HologramManager.java:70) ~[?:?]
4 | at com.booksaw.betterTeams.integrations.HologramManager.createHolo(HologramManager.java:57) ~[?:?]
5 | at com.booksaw.betterTeams.integrations.HologramManager.<init>(HologramManager.java:34) ~[?:?]
6 | at com.booksaw.betterTeams.Main.updateHolos(Main.java:279) ~[?:?]
7 | at com.booksaw.betterTeams.Main.onEnable(Main.java:197) ~[?:?]
8 | at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[minecraft_server.jar:git-Paper-788]
9 | at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[minecraft_server.jar:git-Paper-788]
10 | at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[minecraft_server.jar:git-Paper-788]
11 | at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:518) ~[minecraft_server.jar:git-Paper-788]
12 | at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:432) ~[minecraft_server.jar:git-Paper-788]
13 | at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:594) ~[minecraft_server.jar:git-Paper-788]
14 | at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:298) ~[minecraft_server.jar:git-Paper-788]
15 | at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1069) ~[minecraft_server.jar:git-Paper-788]
16 | at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:291) ~[minecraft_server.jar:git-Paper-788]
17 | at java.lang.Thread.run(Thread.java:831) [?:?]
18 | [20:44:32] [Server thread/INFO]: [BetterTeams] Disabling BetterTeams v4.1.3
```

**Comment by Aadi-mone** [comment]
thanks.
for the great support, I've rated again in spigot!!

**Comment by booksaw** [comment]
This bug has been fixed and the patch will be included in the next update. Thank you for reporting.

I cannot reproduce this bug, so instead, I have detected when it's occurring and fixing the problem on the fly, there is a debug message which looks like this:
`A team was null for an unexplained reason, team name: teamName`
if you start seeing this let me know as it will help point towards the actual bug.
